ClothesApp/ClothesApp/Program.cs

```diff
@@ -1 +1,225 @@
-﻿Console.WriteLine("Hello");
\ No newline at end of file
+﻿using System.Collections;
+using ClothesApp;
+using ClothesApp.Entities;
+using Microsoft.EntityFrameworkCore;
+
+using var db = new ClothesAppContext();
+
+Console.WriteLine("-----------------LAZY-----------------------");
+Console.WriteLine("get_products_by_brand_id");
+var products = await db.Products
+    .Where(p => p.BrandId == 1)
+    .ToListAsync();
+    
+Console.WriteLine("after query string");
+
+foreach (var p in products)
+{
+    Console.WriteLine($"id: {p.Id}, " +
+                      $"brand_id: {p.BrandId}, " +
+                      $"brand_name: {p.Brand.Name}, " +
+                      $"category_id: {p.CategoryId}," +
+                      $"name: {p.Name}," +
+                      $"price: {p.Price}," +
+                      $"quantity: {p.Quantity}," +
+                      $"created at: {p.CreatedAt}");
+}
+    
+Console.WriteLine("---------------------------------------------------------");
```

Comment:
I don't need all these console writeline info, I just need you to show generated ef queries